### PR TITLE
Properly normalize edisp in lstchain v0.9

### PR DESCRIPTION
**Pull request by morcuended**
see #1160

### CODE REVIEW

**Comment by maxnoe** [comment]
I think the interpolation code also needs an update, right? Or don't we have any interpolation code in 0.9.x?

**Comment by maxnoe** [comment]
Ok, interpolation only introduced in 0.10.x, good

File: lstchain/tools/lstchain_create_irf_files.py [hunk 1]
```diff
@@ -569,11 +569,13 @@ def start(self):
...
+        # This normalization definition is assumed in pyirf >= v0.10.0.
         # See https://github.com/cta-observatory/pyirf/pull/250
-        self.log.warning(
-            "Fixing EDISP normalization for v0.9 data products. The proper "
-            "normalization is assumed in lstchain v0.10.5 which uses pyirf >= v0.10.0"
-        )
+        edisp_header = fits.Header()
+        edisp_header["FIXEDNRM"] = True
+        for k, v in extra_headers.items():
+            edisp_header[k] = v
```
Comment (maxnoe) [inline lstchain/tools/lstchain_create_irf_files.py:577]:
```suggestion
        edisp_header = extra_headers.copy()
        edisp_header["FIXEDNRM"] = True
```

File: lstchain/tools/lstchain_create_irf_files.py [hunk 2]
```diff
@@ -569,11 +569,13 @@ def start(self):
...
+
```
Comment (maxnoe) [inline lstchain/tools/lstchain_create_irf_files.py:578]:
```suggestion
```

File: lstchain/tools/lstchain_create_irf_files.py [hunk 3]
```diff
@@ -584,7 +586,7 @@ def start(self):
                 fov_offset_bins,
                 point_like=self.point_like,
                 extname="ENERGY DISPERSION",
-                **extra_headers,
+                **edisp_header,
```
Comment (maxnoe) [inline lstchain/tools/lstchain_create_irf_files.py:589]:
```suggestion
                **extra_headers,
                FIXEDNRM=True,
```

File: lstchain/tools/lstchain_create_irf_files.py [hunk 4]
```diff
@@ -567,6 +567,15 @@ def start(self):
             fov_offset_bins,
             migration_bins,
         )
+        # Different normalization of EDISP had been assumed so far (to the sum of 1).
+        # According to GADF definition, it should be normalized to the integral of 1.
+        # See https://github.com/cta-observatory/pyirf/pull/250
+        self.log.warning(
```
[the remark this answers is not in the document]
Comment (morcuended) [reply lstchain/tools/lstchain_create_irf_files.py]: I removed the warning

**Comment by morcuended** [comment]
I changed the code as suggested. Any more comments @maxnoe?

**Review by maxnoe — APPROVED** [review]

**Pull request merged**